templates/AureliaSpa/webpack.config.js

```diff
@@ -0,0 +1,74 @@
+var isDevBuild = process.argv.indexOf('--env.prod') < 0;
+var path = require('path');
+var webpack = require('webpack');
+var AureliaWebpackPlugin = require('aurelia-webpack-plugin');
+const srcDir = path.resolve('./ClientApp');
+const rootDir = path.resolve();
+const outDir = path.resolve('./wwwroot/dist');
+const baseUrl = '/';
+var project = require('./package.json');
+
+const aureliaBootstrap = [
+    'aurelia-bootstrapper-webpack',
+    'aurelia-polyfills',
+    'aurelia-pal-browser',
+    'regenerator-runtime',
+];
+const aureliaModules = Object.keys(project.dependencies).filter(dep => dep.startsWith('aurelia-'));
+
+// Configuration for client-side bundle suitable for running in browsers
+var clientBundleConfig = {
+     resolve: { extensions: [ '.js', '.ts' ] },
+     devtool: isDevBuild ? 'inline-source-map' : null,
+     entry: {
+        'app': [], // <-- this array will be filled by the aurelia-webpack-plugin
+        'aurelia-bootstrap': aureliaBootstrap,
+        'aurelia-modules': aureliaModules.filter(pkg => aureliaBootstrap.indexOf(pkg) === -1)
```

Comment:
Why are these dependencies going into a separate bundle? Couldn't they just be included in `aurelia-bootstrap`? (which they would be by default if this line wasn't here)